### DISCUSSION TITLE
@TomTravis

### DISCUSSION THREAD

**Discussion by TomTravis** [dnschneid/crouton#4745] [Ideas]
@TomTravis 

Thanks, but no thanks.

-**DennisLfromGA**

_Originally posted by @DennisLfromGA in https://github.com/dnschneid/crouton/issues/4738#issuecomment-1133034040_

try it you like it much better then a [startup script](https://github.com/dnschneid/crouton/blob/master/test/daemon.sh)
[manage moderate](https://salsa.debian.org/cgzones/systemd)
[Crazy](https://www.congress.gov/member/marjorie-greene/G000596)

**Comment by DennisLfromGA** [comment]
Actually, I'm perfectly happy with my chroots as they are, they work great.
```
----------------------------------------------------------------------
chroot name: bionic
crouton: version 1-20220111101058~master:a9eafaee
release: bionic
architecture: amd64
xmethod: xiwi
targets: xiwi,xorg,cli-extra,keyboard,touch,gnome-desktop,lxde-desktop,chrome-beta
host: version 14695.55.0 (Official Build) beta-channel eve 
kernel: Linux localhost 5.4.188-18437-g70f97a1a42f6 #1 SMP PREEMPT Tue May 10 16:07:25 PDT 2022 x86_64 x86_64 x86_64 GNU/Linux
freon: yes
----------------------------------------------------------------------
chroot name: focal
crouton: version 1-20220111101058~master:a9eafaee
release: focal
architecture: amd64
xmethod: xiwi
targets: xiwi,xorg,cli-extra,keyboard,touch,gnome-desktop,lxde-desktop,chrome-beta
host: version 14695.55.0 (Official Build) beta-channel eve 
kernel: Linux localhost 5.4.188-18437-g70f97a1a42f6 #1 SMP PREEMPT Tue May 10 16:07:25 PDT 2022 x86_64 x86_64 x86_64 GNU/Linux
freon: yes
----------------------------------------------------------------------
chroot name: xenial
crouton: version 1-20220111101058~master:a9eafaee
release: xenial
architecture: amd64
xmethod: xiwi
targets: xiwi,xorg,cli-extra,keyboard,touch,gnome-desktop,lxde-desktop,unity-desktop,xfce-desktop,chrome-beta
host: version 14695.55.0 (Official Build) beta-channel eve 
kernel: Linux localhost 5.4.188-18437-g70f97a1a42f6 #1 SMP PREEMPT Tue May 10 16:07:25 PDT 2022 x86_64 x86_64 x86_64 GNU/Linux
freon: yes
----------------------------------------------------------------------
```

**Comment by TomTravis** [comment]
see DEV progress is happening now. Linux kernels are compiling software is being updated

maybe you can "support this" [Confused?](https://man7.org/linux/man-pages/man1/systemctl.1.html)
the command is no longer chroot ignored

z@localhost:~/systemd/src/systemctl$ cd ..
z@localhost:~/systemd/src$ systemctl
System has not been booted with systemd as init system (PID 1). Can't operate.
Failed to connect to bus: Host is down

lol but it is now running? This will be reported 

i hope this help

**Comment by TomTravis** [comment]
Well [systemd](http://systemd.io/) is on hold for now, [Supervisord](http://supervisord.org/) is installed [cron](https://en.wikipedia.org/wiki/Cron) [vsftp](https://en.wikipedia.org/wiki/Vsftpd) [network-manager](https://wiki.gnome.org/Projects/NetworkManager) [rsyslog](https://www.rsyslog.com/) and a few other are now working. I suppose when my [Xorg](https://www.x.org/wiki/) server arrives thing can happen easier.


****************I hope this helps****************** @DennisLfromGA ++++++++++++confused

**Comment by DennisLfromGA** [comment]
@TomTravis,

I don't mean to be rude but I'm actually not interested in these endeavors, as I said, I'm happy with crouton as is.
I'd like to opt out of any further discussion(s) or issue(s) so please exclude me in any further correspondence.

Thanx,
-**DennisLfromGA**
